modules/phase_field/src/base/PhaseFieldApp.C

```diff
@@ -293,6 +294,7 @@ PhaseFieldApp::registerObjects(Factory & factory)
   registerInitialCondition(SpecifiedSmoothCircleIC);
   registerInitialCondition(ThumbIC);
   registerInitialCondition(Tricrystal2CircleGrainsIC);
+  registerInitialCondition(RampIC);
```

Comment:
The include and register statements are sorted alphabetically. Let's keep it that way.